src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background-color: $standart-color;
+  border: 1px solid $card-border-color;
+  border-radius: $main-border-radius;
+
+  padding: 0 16px 16px;
+
+  &-container {
+    display: flex;
+    flex-wrap: wrap;
+    align-content: center;
+    padding: 50px 0;
+    max-width: containerWidth($card-width, $card-margin-right);
+  }
+
+  &__image {
+    margin: 32px 3px 40px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: $subtitle-text-color;
+  }
+
+  &__stars {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__stars-reviews {
+    color: $main-text-color;
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    line-height: 18px;
+
+    &-text {
+      color: $subtitle-text-color;
+      font-size: 12px;
+      font-weight: 400;
+    }
+
+    &-price {
+      font-weight: 700;
+      font-size: 16px;
+      color: $main-text-color;
+    }
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    height: 40px;
+
+    @include textDecorationUpperStandart;
+    font-size: 14px;
+    font-weight: 700;
+    color: $standart-color;
+
+    background-color: $main-color;
+    border: 1px solid $main-color;
+    border-radius: $main-border-radius;
+
+    &:hover {
+      background-color: $standart-color;
+      color: $main-color;
+    }
+  }
+
+  &--margin {
+    margin-right: $card-margin-right;
+    margin-top: $card-margin-top;
+
+    width: $card-width;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-top: 0;
+    }
+  }

```

Comment:
No external positioning for BEM blocks, event through modifiers, move to element and mix to card